lib/checks/shared/non-empty-if-present.js

```diff
@@ -3,7 +3,9 @@ let nodeName = node.nodeName.toUpperCase();
 let type = (node.getAttribute('type') || '').toLowerCase();
 let label = node.getAttribute('value');
 
-this.data(label);
+if (label) {
```

Comment:
The message for this check used the existence of a label to determine the output, which doesn't work with the current schema. So I updated it since the data only needed to know a label was present and not what it was.